compiler/crates/relay-schema/src/relay-extensions.graphql

```diff
@@ -176,7 +176,7 @@ declare how null values should be handled at runtime. You can think of it as
 saying "if this field is ever null, its parent field is invalid and should be
 null".
 
-[Read More](https://www.internalfb.com/intern/staticdocs/relay/docs/guides/required-directive/) (FB only)
+[Read More](https://relay.dev/docs/guides/required-directive/)
```

Comment:
Feel free to undo, if there is still FB specific information on this page, but considering that it's documented and used in OSS I thought it should use the link from the official docs.